src/app.js

```diff
@@ -23,9 +23,10 @@ const App = () => {
             <LocationProvider>
               <Layout>
                 <Router>
-                  <Home path="/frontend/react" />
+                  <Home path="/frontend/react/home" />
```

Comment:
Nit, what are your thoughts on calling what we currently have as `home` about? We don't really have a "home" page/component since the default page is search and the information rendered here is more of an about style snippet?

Reply:
Good question. My initial thought is we can call the previous homepage 'About HeLx' page and change menu name as well. Unsure if that's appropriate, but can discuss that tomorrow